docs/performance/inserts/methods.rst

```diff
@@ -123,6 +126,15 @@ For example, using the CrateDB Python client, the following is possible:
 Here, you can vary the number of rows being inserted without having to change
 the prepared statement.
 
+.. warning::
+    When inserting using ``UNNEST``, e.g. using the Python client, rows that
```

Comment:
I would put the e.g. inside parentheses here

I think this sentence could also benefit from being rewritten to use the active voice

Reply:
I'd agree, except for the 'drop' verb: that refers to unintended behavior, which feels more natural put in passive tense. Alternatively you'd get "rows... may drop", which sounds odder to me.

Reply:
in this case, the doer is CrateDB, so I would imagine it more like "CrateDB may drop rows when...". what do you think?

Reply:
That's a good idea!

Reply:
just looking at this again and reviewing the note in context. I think we can drop the bit about the "Python client". the behavior being described is not specific to the Python client. the only bit which is specific to the Python client is the code example above but I don't think that matters for this warning